graphql-java-support/src/main/java/com/apollographql/federation/graphqljava/FederationDirectives.java

```diff
@@ -28,136 +29,89 @@
 public final class FederationDirectives {
     /* Directive locations */
 
+    public static final String keyName = "key";
+    public static final String externalName = "external";
+    public static final GraphQLDirective external = newDirective()
+            .name(externalName)
+            .validLocations(FIELD_DEFINITION)
+            .build();
+
+    /* fields: _FieldSet */
+    public static final String requiresName = "requires";
+    public static final String providesName = "provides";
+    public static final String extendsName = "extends";
+
+    /* directive @key(fields: _FieldSet!) on OBJECT | INTERFACE */
+    public static final GraphQLDirective extends_ = newDirective()
+            .name(extendsName)
+            .validLocations(OBJECT)
+            .build();
+    public static final Set<GraphQLDirective> allDirectives = new HashSet<>();
+    public static final Set<SDLDefinition> allDefinitions = new HashSet<>();
     private static final DirectiveLocation DL_OBJECT = newDirectiveLocation()
             .name("OBJECT")
             .build();
 
+    /* directive @external on FIELD_DEFINITION */
```

Comment:
Something is wonky here? This comment is referring to the `@external` directive, but the below line is on the `@extends` directive.